nullaway/src/main/java/com/uber/nullaway/dataflow/DataFlow.java

```diff
@@ -63,6 +63,8 @@
    */
   private static final int MAX_CACHE_SIZE = 50;
 
+  private boolean assertsEnabled;
```

Comment:
Let's make this `final`, initialize in the constructor, and get rid of the `setAssertionsEnabled()` method